modules/document_repository/php/files.class.inc

```diff
@@ -144,6 +139,25 @@ class Files extends \NDB_Page
                 ->withHeader("Allow", "GET,POST,PUT,DELETE");
         }
     }
+    /**
+     * Handles the document download process
+     *
+     * @param string $file the file name
+     *
+     * @return void
+     */
+    function download(string $file): void
+    {
+        header('Content-Description: File Transfer');
```

Comment:
This is a big step backwards.. we shouldn't be taking something that's already returning a PSR7 response and regressing to using side-effects and exit. If there's a problem with the withBody() part of the response, that should be fixed